config/validator/service/proposer-settings.go

```diff
@@ -77,3 +123,81 @@ type ProposerOption struct {
 	FeeRecipientConfig *FeeRecipientConfig
 	BuilderConfig      *BuilderConfig
 }
+
+// Clone creates a deep copy of the proposer settings
```

Comment:
Are these clone behaviors tested somewhere?

Reply:
It's tested in the db tests but I'm working on adding very specific unit tests right now for clone.